training.md

```diff
@@ -36,45 +36,123 @@ There may be an issue where your terminal can't find Git or Anaconda, in which c
-   2. `pip install dbt-sqlserver`
-   3. `dbt debug` it should show all green and get no error message!
-
-
-### Exercises
-
-#### Create tables (dbt run)
-
-Let's make some dbt models! The following steps will show how `dim_customers`, a finished table gets created and transformed from raw data sources. We currently already have some source tables in our database: `raw.customers`, `raw.orders`, and `raw.payments` (view these tables in Azure Data Studio under the Tables folder - try to `Select Top 1000`)
-1. We need to create staging tables from the tables listed above. The SQL files that create these tables are located in `models\staging`. Take a quick look at the code to understand the transformations! In the command line, individually run:
+1. Clone [this repo](https://github.com/dbt-msft/jaffle_shop_mssql) in VSCode
+   1. Open VSCode and type (`CTRL + Shift + P`) to access the command palette located at the top.
+   2. Type `git: clone` and select the option.
+   3. Paste the git URL: `https://github.com/dbt-msft/jaffle_shop_mssql` and hit enter.
+   4. Choose which folder you want your code to go in and click "Select Repository Location".
+   6. A pop up message should appear on the lower right of VSCode asking if you would like to open the cloned repository. Click "Open".
+2. After cloning, you should be prompted to install some extensions -- you should! The extensions are:
+   * `python extension`
+   * `better jinja`
+   * `vscode-dbt`
+   * `rainbow csv`
+3. Now we need to set up `profiles.yml` to connect to our database. In our repo, this is called [`profiles_example.yml`](profiles_example.yml).
+   1. Create new directory called `.dbt` under your user folder (`C:\Users\your_user_folder`).
+   2. Open `profiles_example.yml`.
+   3. Save As the file into the `.dbt` directory you just made and rename it to `profiles.yml`.
+   4. Edit the file in VSCode and change the `schema` field on line 5 to your initials or anything that serves as an identifier for your work.
+4. Create your conda environment and install the necessary packages in here **once** so that you can reuse this environment for future dbt work.
+   1. Open a new terminal by clicking on "Terminal" at the top and "New Terminal"
+   2. Click "Allow" when a pop up appears asking if you allow the workspace to modify your terminal shell.
+      * If you see "PS" next to your current directory on the command line, that means you're using powershell and we don't want to use that. The fix: open a new terminal again and you should now be using cmd.
+   3. Since you already installed Anaconda, you should see `conda activate base` run automatically and (base) should be next to your current directory.
+   4. Create a new, empty environment by running `conda create -n dbtenv python=3.7.9`. The name of your environment is `dbtenv` and we are using Python version 3.7.9.
+      * *do we want to include this or should we just have them install in the base environment?*
```

Comment:
we discussed leaving as is for now, but maybe observing users go through tutorial later.
```suggestion
```